src/components/Music/Song.js

```diff
@@ -0,0 +1,58 @@
+import React, { useState, useEffect } from 'react';
+import { useSelector } from 'react-redux';
+import {
+  FaPlay, FaPause, FaStar, FaRegStar,
+} from 'react-icons/fa';
+import '../../styles/Song.css';
+
+export default function Song() {
+  const music = useSelector((state) => state.music);
+  const audio = new Audio(music.music.preview);
+  const [clicked, setClicked] = useState(false);
+
+  const play = () => {
+    audio.play();
+  };
+
+  const pause = () => {
+    audio.pause();
+  };
+
+  const saveSong = () => {
+    if (clicked) {
+      localStorage.removeItem(music.music.title);
+      setClicked(false);
+    } else {
+      localStorage.setItem(music.music.title, JSON.stringify(music.music));
+      setClicked(true);
+    }
+  };
+
+  useEffect(() => {
+    play();
+    return () => {
+      pause();
+    };
+  }, [audio, music]); // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
- [ ] It will be great to fix the eslint error. Disabling eslint rules is advised against since eslint is used to help you write clean code. Kindly remove this disabling rule and fix the error instead.